src/modality-specific-files/positron-emission-tomography.md

```diff
@@ -73,10 +73,43 @@ and a guide for using macros can be found at
    suffixes=["pet", "events", "physio", "stim"])
 }}
 
-PET data MUST be stored in the `pet` directory.
-PET imaging data SHOULD be stored in 4D (or 3D, if only one volume was acquired)
-NIfTI files with the `_pet` suffix.
-Volumes MUST be stored in chronological order (the order they were acquired in).
+### PET data format
+
+All PET imaging data MUST be stored using the
+[NIfTI-1 or NIFTI-2](https://nifti.nimh.nih.gov/) file format (`.nii`).
+The ANALYZE-7.5 file format (`.hdr`/`.img`), despite being very similar to NIFTI-1,
```

Comment:
Not sure if it's still needed. Is `.hdr/.img` popular in PET?

Reply:
No idea but I am starting to think we shold not mention things that are NOT allowed, because it could get very long.

Reply:
@mnoergaard might know :-)